### DISCUSSION TITLE
i18n: Thai translation added, fixed typo in the tk-TM translation file name

### DISCUSSION THREAD

**Discussion by ppnplus** [open-webui/open-webui#4090] [Show and tell]
Hello everyone! I'm a new contributor to Open WebUI. I'd like to discuss changes I'm proposing to the project.

### Description

- In the pull request #4091, I've added the translation for Thai language, although not yet complete, which would benefit potential Thai users using and managing local LLM via Open WebUI.
- In addition to the Thai translation, I've fixed a typo in the translation file name for tk-TM language.

### Added

- Thai translation

### Fixed

- Typo in the translation file name for tk-TM language

**Comment by ppnplus** [comment]
I just spotted another recently merged PR (#4030) also about Thai localization. I've closed my pull request, for now.

**Comment by alvarellos** [comment]
Hi ppnplus, I have added Support for Multi Language for the Openwebui. It might be interesting for you.

Here is the discussion
[[Amber](https://github.com/amberyufangchiu)](https://github.com/open-webui/open-webui/discussions/15839)
and the PR
https://github.com/open-webui/open-webui/pull/15840

I would appreciate to have a feedback :)

Thanks !!
